include/functions.h

```diff
@@ -270,6 +270,24 @@ s32 test_ray_zones(f32 startX, f32 startY, f32 startZ, f32 dirX, f32 dirY, f32 d
                    f32* hitDepth, f32* nx, f32* ny, f32* nz);
 s32 test_ray_colliders(s32 ignoreFlags, f32 startX, f32 startY, f32 startZ, f32 dirX, f32 dirY, f32 dirZ, f32* hitX,
                        f32* hitY, f32* hitZ, f32* hitDepth, f32* hitNx, f32* hitNy, f32* hitNz);
+
+/// Test a general ray from a given starting position and direction against all entities.
+/// If one is hit, returns the position and normal of the hit and the length along the ray on the output params.
+/// All output params are only valid when a value of `NO_COLLIDER` is returned.
```

Comment:
I may be misunderstanding but I'd expect the inverse to be true? That is I'd expect the output params are only valid when `NO_COLLIDER` is NOT returned

Reply:
Oops forgot a word there